contracts/EscrowAndStaking/StakingThales.sol

```diff
@@ -98,9 +99,9 @@ contract StakingThales is IStakingThales, Initializable, ProxyOwned, ProxyReentr
 
     uint public totalStakedLastPeriodEnd;
     uint public totalEscrowedLastPeriodEnd;
-    address public exoticBonds;
+    address private exoticBonds;
 
-    IAddressResolver public addressResolver;
+    IAddressResolver private addressResolver;
 
     address public thalesRangedAMM;
     address public sportsAMM;
```

Comment:
add to addressMAnager and use it from there. Make this variable private.

Reply:
AddressManager will be used as AddressResolver in Staking contract